AltStore/Server/ServerManager.swift

```diff
@@ -140,7 +140,7 @@ private extension ServerManager
     func addDiscoveredServer(_ server: Server)
     {
         var server = server
-        server.isPreferred = (server.identifier == UserDefaults.standard.preferredServerID)
+        server.isPreferred = true
```

Comment:
Was this intentional?

Reply:
Yes, I didn’t realize this was in this commit but my thinking was that if AltStore finds a local server it should prefer it over the VPN AltServer.